.github/workflows/build-checking.yml

```diff
@@ -0,0 +1,37 @@
+name: Check Build
+
+env:
+  DRAGONFLY_PATH: "${{github.workspace}}/build-opt/dragonfly"
+  
+on: 
+  schedule:
+    - cron: '0 7 */3 * *'
+  workflow_dispatch:
+      
+jobs:
+  build-n-test:
+    runs-on: ubuntu-dev:20
```

Comment:
hmm, are you sure this works? did you actually try it?
I meant using container image directive : https://github.com/dragonflydb/dragonfly/blob/main/.github/workflows/ci.yml#L44